GitUI/CommandsDialogs/WorktreeDialog/FormManageWorktree.cs

```diff
@@ -207,8 +207,8 @@ private void Worktrees_CellClick(object sender, DataGridViewCellEventArgs e)
 
             if (e.ColumnIndex == 5)
             {
-                if (MessageBox.Show(this, "Are you sure you want to switch to this worktree?", "Open a worktree",
-                    MessageBoxButtons.YesNo) == DialogResult.Yes)
+                if (AppSettings.DontConfirmSwitchWorktree || MessageBox.Show(this,
+                        "Are you sure you want to switch to this worktree?", "Open a worktree", MessageBoxButtons.YesNo) == DialogResult.Yes)
```

Comment:
Could you have the kindness to translate these strings? I know it is not your fault but it is the good moment to do it...

Reply:
Sure, I gave it a try ;)